rpm/lxc.spec

```diff
@@ -18,7 +19,7 @@
 %global shortcommit %(c=%{commit}; echo ${c:0:7})
 
 Name:           lxc
-Version:        3.0.1
+Version:        4.0.10
 Release:        1
 Summary:        Linux Resource Containers
 Group:          Applications/System
```

Comment:
Please remove Group tags.

Reply:
Also change "%setup -q -n %{name}-%{version}/%{name}" to "%autosetup -p1 -n %{name}-%{version}/%{name}"